users/ignoring.rst

```diff
@@ -88,6 +88,32 @@ The ``.stignore`` file contains a list of file or path patterns. The
    are *included* (that is, *not* ignored). This can be used to override
    more general patterns that follow.
 
+   .. note::
+
+      Negated patterns that can match items below the folder root will cause
+      Syncthing to traverse otherwise ignored directories. If the
+      :ref:`watcher <scanning>` is enabled, those directories will also be
+      watched. Directories ignored before the first negated pattern can
+      however be safely skipped, since the first matching pattern wins. For
+      example::
+
+         /foo
+         /bar
+         !baz
+         *
+
+      The directories ``foo`` and ``bar`` will be entirely ignored. However any
+      other directories present must be scanned entirely to find any items
+      named `baz`, despite the fact that they will be ignored due to the
```

Comment:
Also a difficult sentence, should be split up at the comma and refactored.  Speaking names in the example will help to reference easily.  Include a number to avoid repeating long names here.